frontend/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/annotations.e2e.js.md

```diff
@@ -1007,7 +1007,7 @@ Generated by [AVA](https://ava.li).
       ],
       task: {
         boundingBox: null,
-        created: 1477666728000,
+        created: 'created',
```

Comment:
do we know what happened to the snapshots here?

Reply:
Yes. It happens because we now call the `replaceVolatileValues` function for annotation objects. This was necessary because we update the modified timestamps of annotation when finishing them.